salt/cloud/clouds/vmware.py

```diff
@@ -2996,7 +2996,7 @@ def create_folder(kwargs=None, call=None):
         return {inventory_path: 'created the specified path'}
 
 
-def create_snapshot(name, kwargs=None, call=None):
+def create_snapshot(name=None, kwargs=None, call=None):
```

Comment:
Why is ``name=None`` required? For actions, name of the VM always has to be specified otherwise the VM lookup will fail. All other actions are the same way.